packages/reason-parser/reason-parser.2.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Jordan Walke <jordojw@gmail.com>"
+authors: "Jordan Walke <jordojw@gmail.com>"
+homepage: "https://github.com/facebook/reason"
+bug-reports: "https://github.com/facebook/reason/issues"
+license: "BSD"
```

Comment:
Would it please be possible to clarify that the license is strictly speaking not BSD. 

With the advent of opam local switches and `odig` you can quickly find out what licenses your app depends on  by doing `odig license -t --all --warn-error | sort | uniq`. This certainly won't give a you a formal legal investigation but it's good if we can make that lie as little as possible.